lib/yi_appium_caps_util.rb

```diff
@@ -181,11 +187,23 @@ def update_ios_caps (output_data)
         elsif (xcodeBuildVersion.to_f>8) then
             puts "Change to xcode version to xcode 7! Current version is: "+xcodeBuildVersion
         end
+      end  
+    rescue Exception => ex
+      puts "An error of type #{ex.class} happened, message is #{ex.message}"
+      raise
     end
 
+    def update_mac_caps (output_data)
+      #Replace value of deviceName
+      output_data['caps']['deviceName'] = "macOS device"
+      #Replace value of youiEngineAppAddress
+      output_data['caps']['youiEngineAppAddress'] = "localhost"
+    rescue RuntimeError => ex
+      puts ex.message
+      raise
     rescue Exception => ex
       puts "An error of type #{ex.class} happened, message is #{ex.message}"
-      exit
+      raise
```

Comment:
you can just `riase 'An error of type #{ex.class} happened, message is #{ex.message}'` instead over here

Reply:
This also goes for all the other raises before